app/src/test/java/com/codesoom/assignment/domain/CatToyRepositoryTest.java

```diff
@@ -0,0 +1,194 @@
+package com.codesoom.assignment.domain;
+
+import com.codesoom.assignment.dto.CatToyNotFoundException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("CatToyRepositoryTest 클래스")
+class CatToyRepositoryTest {
+
+    private CatToyRepository catToyRepository;
+    private CatToy catToy1;
+    private CatToy catToy2;
+    private CatToy catToy3;
+
+    @BeforeEach
+    void setUp() {
+        catToy1 = new CatToy(1L, "toy1", "maker1", 1000L, "toy1.jpg");
+        catToy2 = new CatToy(2L, "toy2", "maker2", 2000L, "toy2.jpg");
+        catToy3 = new CatToy(3L, "toy3", "maker3", 3000L, "toy3.jpg");
+
+        catToyRepository = new CatToyRepository();
+    }
+
+    @Nested
+    @DisplayName("save 메소드")
+    class Describe_save {
+
+        @Test
+        @DisplayName("catToy가 Repository에 추가됩니다.")
```

Comment:
DisplayName의 설명을 쭉 이어서 읽어보세요. 어느 쪽이 좀 더 완성된 문장인가요?

- "CatToyRepositoryTest 클래스의 save 메소드는, catToy가 repository에 추가됩니다."
- "CatToyRepository 클래스의 save 메소드는 catToy를 repository에 저장하고, 저장된 catToy를 리턴합니다."

```suggestion
        @DisplayName("catToy를 repository에 저장하고, 저장된 catToy를 리턴합니다.")
```